core/src/main/java/com/linecorp/armeria/internal/server/annotation/AnnotatedServiceFactory.java

```diff
@@ -737,6 +743,26 @@ static String findDescription(AnnotatedElement annotatedElement) {
                 checkArgument(!value.isEmpty(), "value is empty");
                 return value;
             }
+        } else if (annotatedElement instanceof Parameter) {
+            // JavaDoc/KDoc descriptions only exist for method parameters
+            final Parameter parameter = (Parameter) annotatedElement;
+            final Executable executable = parameter.getDeclaringExecutable();
+            final Class<?> clazz = executable.getDeclaringClass();
+            final String fileName = getFileName(clazz.getCanonicalName());
+            final String propertyName = executable.getName() + '.' + parameter.getName();
+            try (InputStream stream = AnnotatedServiceFactory.class.getClassLoader()
+                                                                   .getResourceAsStream(fileName)) {
+                if (stream == null) {
+                    return null;
+                }
+                final Properties properties = new Properties();
+                properties.load(stream);
+                if (properties.containsKey(propertyName)) {
+                    return properties.getProperty(propertyName);
+                }
+            } catch (IOException exception) {
```

Comment:
Could we cache this so it's not loaded as many times as the number of service methods?